Airports/EGNJ/Fixes.txt

```diff
@@ -1,2 +1,10 @@
 ;Southern North Sea HMR Fix
 LAGER N053.36.40.000 W000.08.49.000
+
+KULOS N053.26.36.250 W000.26.27.860
+NJ02F N053.29.22.510 W000.24.33.520
+NJM01 N053.38.32.130 W000.18.13.610
+OTR   N053.41.53.570 W000.06.13.450
```

Comment:
```suggestion
```
OTR is not a fix, it's the Ottringham VOR, which we include in our VOR definitions :)